src/tangle/sources/mod.rs

```diff
@@ -24,18 +24,26 @@ use crate::types::{
 /// Logical grouping of data that belongs to a milestone.
 #[derive(Clone, Debug)]
 pub struct MilestoneData {
+    /// The [`MilestoneId`] of the milestone.
```

Comment:
These comments provide no benefit. I would rather have a `#[allow(missing_docs)]` tag

Reply:
fair enough ... (there they go ... my 5 mins of hard concentration to not make any typos)